pkg/util/dtotel/otel.go

```diff
@@ -22,7 +22,7 @@ type shutdownFn func(ctx context.Context) error
 func Start(ctx context.Context, otelServiceName string, apiReader client.Reader, webhookNamespace string) func() {
 	endpoint, apiToken, err := getOtelConfig(ctx, apiReader, webhookNamespace)
 	if err != nil {
-		log.Info("failed to read OpenTelementry config secret", "err", err.Error())
+		log.Info("Otel was not configured, add the secret if you want to enable this functionality.")
 
 		return setupNoopOTel()
 	}
```

Comment:
should we have a log as well that indicates that otel was enabled?

Reply:
IMO not necessary